### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by TontonBuck**

### CODE REVIEW

File: Clic&Farm-iOSMockTests/ApolloTests.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import XCTest
+@testable import Clic_Farm_iOS
+import Apollo
+
+class ApolloTests: XCTestCase {
+
+  override func setUp() {
```
Comment (Aquaj) [inline Clic&Farm-iOSMockTests/ApolloTests.swift]: Overriding to call super seems unnecessary. Is this really useful?

File: Clic&Farm-iOSMockTests/ApolloTests.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    super.setUp()
+  }
+
+  override func tearDown() {
+    super.tearDown()
```
Comment (Aquaj) [inline Clic&Farm-iOSMockTests/ApolloTests.swift]: Overriding to call super seems unnecessary. Is this really useful?

File: Clic&Farm-iOSTests/AddInterventionVCTableViewTests.swift [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+  func test_tableViewHeightForRowAndEstimatedHeightForRow_withRandomTableView() {
+    //Given
+    let tableView = UITableView()
+    let indexPath = IndexPath(row: 0, section: 0)
+    let heightForRow = addInterventionVC.tableView(tableView, heightForRowAt: indexPath)
+    let estimatedHeight = addInterventionVC.tableView(tableView, estimatedHeightForRowAt: indexPath)
+
+    //Then
+    XCTAssertEqual(heightForRow, 60, "Height should be 60")
```
Comment (Aquaj) [inline Clic&Farm-iOSTests/AddInterventionVCTableViewTests.swift]: Are all those Height/EstimatedHeight tests testing the intervention's presence or are they testing the actual layout?
If it is the layout this isn't a great idea, it is better to test functionality, layouts/UIs might change.

File: Clic&Farm-iOSTests/SeedCreationViewTests.swift [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    super.tearDown()
+  }
+
+  func test_textFieldShouldReturn_shouldReturnFalse() {
+    // Given
+    let textField = UITextField()
+
+    // Then
+    XCTAssertFalse(addInterventionVC.inputsSelectionView.seedView.textFieldShouldReturn(textField),
+                  "Should return false")
```
Comment (Aquaj) [inline Clic&Farm-iOSTests/SeedCreationViewTests.swift]: "Should return false" doesn't really inform us about the expected behaviour. Why should it return `false`, what does it mean when it returns true?

File: Clic&Farm-iOSMockTests/ApolloTests.swift (hunk 1 above)
Comment (TontonBuck) [reply Clic&Farm-iOSMockTests/ApolloTests.swift]: We didn't make Apollo tests yet, that's why this class is useless. Removed mock tests from project.

**Pull request merged**